server/src/models/Milestone/milestone.routes.ts

```diff
@@ -5,9 +5,16 @@ const router: Router = Router();
 // @ts-ignore missing correct dependency injection
 const controller = new MilestoneController();
 
-router.post('/milestone', controller.createMilestone);
-router.get('/milestone/:milestoneId', controller.getAllTasksInMilestone);
+router.get('/milestone/sse', controller.milestoneSSE); // connect to client
+router.post('/milestone', controller.createMilestone, controller.milestoneSSE);
```

Comment:
maybe the way that we are connecting the client and the post method are not correct.